libflux/flux-core/src/formatter/mod.rs

```diff
@@ -372,9 +432,23 @@ impl Formatter {
         }
     }
     fn format_record_type(&mut self, n: &ast::RecordType) {
-        let multiline = n.properties.len() > 4 || n.base.is_multiline();
+        let mut multiline = n.properties.len() > 4;
```

Comment:
Maybe this `4` should be defined as a constant in this file? (Assuming that 4 is the value that's used everywhere)